### DISCUSSION TITLE
Problème avec Mailu et certificat Let's Encrypt

### DISCUSSION THREAD

**Discussion by AlainCharell** [Mailu/Mailu#3316] [User support]
Bonjour,

Je rencontre des difficultés pour configurer Mailu avec un certificat Let's Encrypt sur mon serveur. Voici les points clés du problème :

Certificat Let's Encrypt :

Les tentatives de renouvellement du certificat échouent avec l'erreur "too many certificates already issued".
Le challenge Let's Encrypt retourne une erreur 404 pour les fichiers de validation.
Configuration Nginx :

J'ai configuré Nginx pour rediriger les requêtes vers Mailu mais je rencontre des erreurs indiquant que l'upstream "mailu_front_1" n'est pas trouvé.
La configuration actuelle est la suivante pour le certificat :

server {
    listen 80;
    server_name mail.konex-business.fr;
    location /.well-known/acme-challenge/ {
        root /var/www/letsencrypt;
    }
}
Logs d'erreur Nginx :

Les erreurs spécifiques rencontrées dans les logs incluent :

2024/06/25 17:21:58 [emerg] 6281#6281: host not found in upstream "mailu_front_1" in /etc/nginx/sites-enabled/mail.konex-business.fr:20
2024/06/25 17:36:33 [error] 6948#6948: *95 open() "/var/www/letsencrypt/.well-known/acme-challenge/SnAWXpr6w_CatZ8aKFRUwlomRGrgfoH15OhEoWJ1eAQ" failed (2: No such file or directory), client: 23.178.112.206, server: mail.konex-business.fr
Malgré ces efforts, les certificats Let's Encrypt ne sont toujours pas validés correctement et le service Mailu ne fonctionne pas comme attendu. Toute aide ou suggestion pour résoudre ce problème serait grandement appréciée.

Merci.

**Comment by nextgens** [comment]
"too many certificates already issued" -> l'autorité de certification refuse de délivrer de nouveaux certificats: en général il faut juste attendre.

La configuration proposée pour nginx ne fonctionnera pas. Il faut rediriger tout le traffic sur le port 80 pour ``mail.konex-business.fr`` vers Mailu.
